client/components/phone-input/index.jsx

```diff
@@ -243,7 +243,8 @@ class PhoneInput extends React.PureComponent {
 					name={ this.props.name }
 					ref={ this.setNumberInputRef }
 					type="tel"
-					className={ classnames( { 'is-error': this.props.isError } ) }
+					disabled={ this.props.disabled }
+					className={ this.props.isError && 'is-error' }
```

Comment:
It seems that once there's a validation error in the phone field, it stays in the `is-error` state 🤔

Reply:
Definitely 🤔 ... Haven't been able to reproduce this yet. But I'm going to try!